apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -279,15 +279,15 @@ public void addExcludedCollection(StandardCollection collection) {
      * @param collection the StandardCollection to exclude.
      * @see ExclusionProcessor#addFileProcessor(StandardCollection)
      */
-    public void addExcludedFileProcessor(StandardCollection collection) {
+    public void addExcludedFileProcessor(final StandardCollection collection) {
```

Comment:
What to do with unused methods in here ..... (possibly referenced in other PRs) @Claudenw